Kinvey/Kinvey/User.swift

```diff
@@ -538,7 +551,7 @@ open class User: NSObject, Credential, Mappable {
                 micLoginVC.setValue(forceUIWebView, forKey: "forceUIWebView")
             }
             micLoginVC.client = client
-            micLoginVC.micApiVersion = client.micApiVersion
+            micLoginVC.micApiVersion = client.micApiVersion?.rawValue
```

Comment:
micLoginVC is a Obj-C class, which we plan to replace soon for a Swift version of it. So, for this specific PR the rawValue still the best way to pass through.